gradle/verification-keyring.keys

```diff
@@ -14893,3 +14893,107 @@ i8oRVy9MSV3HFJxF7JpNoeEgRBGWCIQJPZ7UZszl4VxSwDkvGoFpZr2MIcXnm++n
 FTmJaS8NbrV1fYW7qPqDNdPbzLO95qRfPIOI
 =FL9o
 -----END PGP PUBLIC KEY BLOCK-----
+
+pub rsa2048/cb3808e03cd602a6dc5b882552931f4b72b4f54c 2014-07-19T14:24:53Z
```

Comment:
These are copied over from https://github.com/gradle/gradle/pull/18003/files

I am not 100% sure what is the proper way of adding these keys and not even sure how to find them.

Reply:
This should be edited by following the instructions in `gradle/verification-metadata.xml`